db/builder.cc

```diff
@@ -61,7 +61,8 @@ TableBuilder* NewTableBuilder(
 Status BuildTable(
     const std::string& dbname, Env* env, const ImmutableCFOptions& ioptions,
     const MutableCFOptions& mutable_cf_options, const EnvOptions& env_options,
-    TableCache* table_cache, InternalIterator* iter, FileMetaData* meta,
+    TableCache* table_cache, InternalIterator* iter,
+    ScopedArenaIterator range_del_iter, FileMetaData* meta,
```

Comment:
fine, just to pass review, not because I agree with explicitly requiring rvalue reference as I explained on the other diff